addon/dev-prefs.json

```diff
@@ -1,5 +1,4 @@
 {
-  "BASE_URL": "http://testpilot.dev:8000",
-  "HOSTNAME": "testpilot.dev",
-  "ALLOWED_ORIGINS": "http://testpilot.dev:8000/*,https://testpilot.firefox.com/*"
+  "toolkit.telemetry.server": "http://127.0.0.1/",
```

Comment:
This may or may not be a good thing, though. I stuck in in there, because along with [gzipServer](https://github.com/vdjeric/gzipServer), this lets me see Telemetry pings go to my local server on `npm run once`